Assets/MixedRealityToolkit.Services/InputSystem/FocusProvider.cs

```diff
@@ -517,11 +517,42 @@ public override void Update()
             if (!IsSetupValid) { return; }
 
             UpdatePointers();
+
+            if (gazeProviderPointingData?.Pointer != null)
+            {
+                UpdateGazeProvider();
+            }
             UpdateFocusedObjects();
 
             PrimaryPointer = primaryPointerSelector?.Update();
         }
 
+        /// <summary>
+        /// Update the hit information for the gaze provider.
+        /// 
+        /// For the sake of StabilizationPlaneModifier and potentially other 
+        /// components that care where the user's looking, we need to do a gaze 
+        /// raycast even if gaze isn't used for focus.
+        /// Assumes gaze provider is not null;
```

Comment:
>; [](start = 45, length = 1)

Nit: . instead of ;

(I'm using codeflow so suggestions aren't possible AFAIK)